cflib/crtp/__init__.py

```diff
@@ -47,8 +47,11 @@
 def init_drivers(enable_debug_driver=False, enable_serial_driver=False):
     """Initialize all the drivers."""
 
-    env = os.getenv('USE_CFLINK')
```

Comment:
What about keeling the env. variable to be able to keep experimenting with links?

If we keep it, I would like to have it trigger an error if the env. variable is set but the value does not correspond to a supported link (right now it defaults silently to python which can be confusing in case of spelling mistake)

Reply:
Sounds nice, like the commit just added?